patches/0006-wic-bootimg-efi-implement-include-path.patch

```diff
@@ -12,14 +12,14 @@ Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
  1 file changed, 4 insertions(+)
 
 diff --git a/scripts/lib/wic/plugins/source/bootimg-efi.py b/scripts/lib/wic/plugins/source/bootimg-efi.py
-index 2cfdc10ecd..fa320a50b8 100644
+index 2cfdc10ecd..3b5a1041a7 100644
 --- a/openembedded-core/scripts/lib/wic/plugins/source/bootimg-efi.py
 +++ b/openembedded-core/scripts/lib/wic/plugins/source/bootimg-efi.py
 @@ -289,6 +289,10 @@ class BootimgEFIPlugin(SourcePlugin):
          mcopy_cmd = "mcopy -i %s -s %s/* ::/" % (bootimg, hdddir)
          exec_native_cmd(mcopy_cmd, native_sysroot)
  
-+        for path in part.include_path:
++        for path in part.include_path or []:
```

Comment:
Why don't we use copyhardlinktree as OE uses in the rootfs plugin?

Reply:
This synax works well to copy files inside fat image.